<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDBusAction</class>
 <widget class="QWidget" name="EditDBusAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeView" name="tvDBusApps"/>
   </item>
   <item row="0" column="1">
    <widget class="QTreeView" name="tvDBusFunctions"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="tvArguments"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
